crates/ruma-client-api/src/membership/mutual_rooms.rs

```diff
@@ -17,36 +17,45 @@ pub mod unstable {
         rate_limited: true,
         authentication: AccessToken,
         history: {
-            unstable => "/_matrix/client/unstable/uk.half-shot.msc2666/user/mutual_rooms/:user_id",
+            unstable => "/_matrix/client/unstable/uk.half-shot.msc2666/user/mutual_rooms",
         }
     };
 
     /// Request type for the `mutual_rooms` endpoint.
     #[request(error = crate::Error)]
     pub struct Request {
         /// The user to search mutual rooms for.
-        #[ruma_api(path)]
+        #[ruma_api(query)]
         pub user_id: OwnedUserId,
+
+        /// An opaque string, used to paginate.
```

Comment:
Something like this would be clearer:

```suggestion
        /// The `next_batch_token` returned from a previous response, to get the next batch of rooms.
```